string agent
string status # {start, done}
string[] task_labels 	
float32[] position
